.changeset/fifty-rabbits-flow.md

```diff
@@ -0,0 +1,17 @@
+---
+'@swisspost/design-system-components': major
+'@swisspost/design-system-components-angular': major
+'@swisspost/design-system-components-react': major
+'@swisspost/design-system-styles': major
+'@swisspost/design-system-migrations': major
+---
+
+Syncing versions of the following packages:
+
+- @swisspost/design-system-styles
+- @swisspost/design-system-components
+- @swisspost/design-system-components-react
+- @swisspost/design-system-components-angular
+- @swisspost/design-system-migrations
```

Comment:
Don't we need to add newly added synced packages (intranet-header, icons) here as well?

Reply:
true, forgot about that